index.html

```diff
@@ -1,21 +1,520 @@
+    <script type="module" src="./scripts/main.js"></script>
+  </head>
+  <body class="bg-secondary-Isabelline">
+    <div
+      class="grid grid-flow-col px-6 md:px-10 lg:px-40 pt-10 laptop-large:px-40"
+    >
+      <ul
+        class="
+          body-copy-1
+          flex
+          items-center
+          leading-4.407
+          opacity-70
+          font-bold
+          text-secondary-san-juan-blue
+        "
+      >
+        <li class="pr-10">
+          <img src="./styles/assets/shared/desktop/logo.svg" alt="" />
+        </li>
+        <li class="hidden md:block pl-5 pr-10"><a href="">Pricing</a></li>
+        <li class="hidden md:block pr-10"><a href="">About</a></li>
+        <li class="hidden md:block"><a href="">Contact</a></li>
+      </ul>
+      <div class="hidden md:flex justify-end items-center">
+        <button class="btn btn--primary">Shedule a Demo</button>
+      </div>
+      <div class="md:hidden flex justify-end items-center">
+        <img src="./styles/assets/shared/mobile/menu.svg" alt="" />
+      </div>
+    </div>
 
-<head>
-  <meta charset="UTF-8" />
-  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Pay API</title>
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link
-    href="https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=Public+Sans:wght@400;700&display=auto"
-    rel="stylesheet">
-  <link rel="stylesheet" href="./styles/tailwind.css">
-  <script type="module" src="./scripts/main.js"></script>
-</head>
-
-<body>
-  <h1 class=" font-public-sans">Hello Alex</h1>
-</body>
-
-</html>
\ No newline at end of file
+    <div
+      class="
+        laptop-large:flex
+        laptop-large:flex-row-reverse
+        laptop-large:items-center
+        laptop-large:justify-center
+      "
+    >
+      <div class="flex justify-center pl-14">
+        <img
+          src="./styles/assets/home/desktop/illustration-phone-mockup.svg"
+          class=""
+          alt=""
+        />
+      </div>
+      <div
+        class="
+          flex flex-col
+          justify-center
+          laptop-large:items-start
+          px-6
+          md:px-24
+          laptop-large:p-0
+          pt-6
+          laptop-large:pt-38 laptop-large:w-136.5
+        "
+      >
+        <p
+          class="
+            heading-3-big
+            leading-9
+            -tracking-0.063
+            text-center
+            font-normal
+            laptop-large:pb-5 laptop-large:text-left
+            md:text-5xl
+            laptop-large:text-7xl
+            md:leading-14
+            laptop-large:leading-18
+            md:-tracking-0.092
+            laptop-large:-tracking-0.138
+            text-secondary-san-juan-blue
+          "
+        >
+          Start building with our APIs for absolutely free.
+        </p>
+        <div class="w-full md:flex md:items-end md:justify-end">
```

Comment:
The `input` and the `button` should be inside a `form` element. The  `button` overflows outside of the input border in tablet and desktop view. It appears contained in the Figma sketch
<img width="312" alt="Screen Shot 2021-11-22 at 8 00 25 PM" src="https://user-images.githubusercontent.com/26441464/142957412-b8783ea6-e5a9-4d7b-aff3-8679b7a30cec.png">

Reply:
size adjusted and box shadow added to input.